lib/internal/test_runner/mock.js

```diff
@@ -346,4 +343,35 @@ function validateTimes(value, name) {
   validateInteger(value, name, 1);
 }
 
+function getOriginalObject({ descriptor, getter, setter }) {
```

Comment:
We should probably rename this to something like `getOriginalFunction()` or `getOriginalMethod()`. I guess technically we are getting a function object, but I think the code would be more readable with a different name.

Reply:
It also looks like this is still the only call site. Why not just have the `if` statement inline.